pkg/console/install_panels.go

```diff
@@ -610,3 +633,61 @@ func addInstallPanel(c *Console) error {
 	installV.Frame = true
 	return nil
 }
+
+func printToPanel(g *gocui.Gui, panel string, message string, clear bool, fgColor gocui.Attribute) {
+	g.Update(func(g *gocui.Gui) error {
+		v, err := g.View(panel)
+		if err != nil {
+			return err
+		}
+		if clear {
+			v.Clear()
+		}
+		v.FgColor = fgColor
+		v.Write([]byte(message))
+		return nil
+	})
+}
+
+func addSyncTaskPanel(c *Console, task AsyncTask) error {
+	maxX, maxY := c.Gui.Size()
+	asyncTaskV := widgets.NewPanel(c.Gui, asyncTaskPanel)
+	asyncTaskV.SetLocation(maxX/4, maxY/4+7, maxX/4*3, maxY/4+9)
+
+	prefix := fmt.Sprintf("%s...", task.Prefix)
+	const infoColor = gocui.ColorCyan
+	const errorColor = gocui.ColorRed
+
+	asyncTaskV.PreShow = func() error {
+		c.Gui.Cursor = false
+		ticker := time.NewTicker(100 * time.Millisecond)
+		go func(g *gocui.Gui) {
+			printToPanel(g, asyncTaskPanel, fmt.Sprintf("%s|", prefix), true, infoColor)
+			for {
+				for _, s := range `\-/|` {
+					<-ticker.C
+					printToPanel(g, asyncTaskPanel, fmt.Sprintf("\r%s%c", prefix, s), false, infoColor)
+				}
+			}
```

Comment:
The loop never breaks

Reply:
Thanks. Fixed.
I also fixed some issues:
- The user still can type during the spinning
- Error message is not displayed if the async task is finished too fast.

I moved the logic back to the "Enter" key event handler, I feel it's more intuitive there.